tests/spatial/affine_test.py

```diff
@@ -65,3 +65,13 @@ def test_can_set_value_of_0d_variable():
     var = affine_transform(value=value)
     var.value += value
     assert np.array_equal(var.value, value + value)
+
+
+def test_inverse():
+    rng = np.random.default_rng()
+    value = rng.random((4, 4))
```

Comment:
Won't this test be flaky due to potentially singular matrices?

Reply:
Good point!